blog/_posts/2023-12-08-migration-to-scala-3.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: blog-detail
+post-type: blog
+by: Zelenya
+title: "Scala 3: My migration journey"
+description: I want to share my journey from Scala 2 to Scala 3 and discuss migration to Scala 3
+---
+
+My name is Jay, I am a software engineer and developer advocate. We've recently migrated the project from Scala 2 to Scala 3 at $WORK, and in this article, I'd like to share the lessons learned and tips.
+
+## Implicit context
+
+Before doing any actual personal work, based on the internet discussions, I had an impression that switching to Scala 3 would be quite painful — in practice, it was pretty straightforward.
+
+In retrospect, our experience was determined by the technology stack and tools that I (and my team) use: microservices, Typelevel stack, VS Code with Metals, and, most importantly, no macros. It might not be as smooth if any of these wasn't the case.
```

Comment:
There is `I` in the beginning to provide context about me and `I` closer to the end for the tooling context. Mixing we/you because the article is about the subjective experience and generic advice.

I see a single personal `I` in the plugin section to `we` that I can change.